msteams-platform/bots/how-to/authentication/add-authentication.md

```diff
@@ -33,7 +33,7 @@ In this article you'll learn:
     | Sample | BotBuilder version | Demonstrates |
     |:---|:---:|:---|
     | **Bot authentication** in [cs-auth-sample][teams-auth-bot-cs] | v4 | OAuthCard support |
-    | **Bot authentication** in [js-auth-sample][teams-auth-bot-js] | v4| OAuthCard support  |
+    | **Bot authentication** in [js-auth-sample](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-teams-authentication/nodejs) | v4 | OAuthCard support |
```

Comment:
Please don't directly add the link here:

There will be a section for `[teams-auth-bot-js]` so please add the link there or please create another section if duplicate.

Reply:
Updated.